docs/windows/policy/tailscale.admx

```diff
@@ -252,5 +256,26 @@
         <string>hide</string>
       </disabledValue>
     </policy>
+    <policy name="ManagedByOrganizationName" class="Machine" displayName="$(string.ManagedByOrganizationName)" explainText="$(string.ManagedByOrganizationName_Help)" presentation="$(presentation.ManagedByOrganizationName)" key="Software\Policies\Tailscale">
+      <parentCategory ref="UI_Category" />
+      <supportedOn ref="SINCE_V1_62" />
+      <elements>
+        <text id="ManagedByOrganizationNamePrompt" valueName="ManagedByOrganizationName" required="true" />
+      </elements>
```

Comment:
It might make sense to group these as one item that has multiple text boxes to fill in, since it seems reasonable to set all of them or none of them.

You can do this by adding extra `<text />` tags inside `<elements></elements>`, rather than 3 separate `<policy></policy>` tags. Maybe relax the `required="true"` though.